README.md

```diff
@@ -99,10 +95,20 @@ development cycle when working on acceptance tests.
 
 	cd repository-data/webfiles
 	mvn com.github.warmuuh:libsass-maven-plugin:0.2.8-libsass_3.4.4:watch
+
+**To provision and Run the local dev server without autoexport**
+
+	make serve.noexport
+
+**To provision and Run the local dev server with autoexport (default)**
+
+	make serve
+
+#### Auto export - on/off
+
+There are 2 ways to run the server: with or without `autoexport`. `autoexport` is switched on by default, and when the app runs like that it export changed and new files triggered by CMS content updates and additions. To avoid these files accidentally being tracked by git, you better run the app without `autoexport`.
```

Comment:
Auto - export is no longer turned on my default.
I've added a description of the up to date behaviour in my PR -> https://github.com/NHS-digital-website/hippo/pull/175/files#diff-04c6e90faac2675aa89e2176d2eec7d8